synfig-studio/src/gui/states/state_bone.cpp

```diff
@@ -217,7 +229,12 @@ StateBone_Context::load_settings()
 		if(settings.get_value("bone.bone_width",value) && value!="")
 			set_bone_width(Distance(atof(value.c_str()),App::distance_system));
 		else
-			set_bone_width(Distance(1,App::distance_system)); // default width
+			set_bone_width(Distance(6.6,App::distance_system)); // default width
+
+		if(settings.get_value("bone.tip_width",value) && value!="")
```

Comment:
prefer `!value.empty()` rather `value!=""`.
Although it normally has the same behavior, if for any reason synfig or glib changes to `std::wstring`, it wouldn't compile. :)
Yes, I know it not probable, but… :)

Reply:
This issue is still present in current code.

Reply:
I cleaned this.